src/govuk/components/checkboxes/checkboxes.js

```diff
@@ -56,11 +51,12 @@ Checkboxes.prototype.syncAll = function () {
 }
 
 Checkboxes.prototype.syncWithInputState = function ($input) {
-  var inputIsChecked = $input.checked
-  $input.setAttribute('aria-expanded', inputIsChecked)
-
   var $content = this.$module.querySelector('#' + $input.getAttribute('aria-controls'))
+
   if ($content) {
```

Comment:
Additional suggestion for increasing consistency: [radios](https://github.com/alphagov/govuk-frontend/blob/7680cb300ffdab8cf5ae7b2e4fdf01fcdcc08d09/src/govuk/components/radios/radios.js#L61) have a check `$content.classList.contains('govuk-radios__conditional')` on this line.

Reply:
Good spot. I think we were going to do this in #1843 but can include that here now instead.